migrations/2021-08-23-161500_aggregated_ft_nft_tables/up.sql

```diff
@@ -0,0 +1,62 @@
+CREATE TABLE aggregated__fungible_tokens
```

Comment:
I feel that "fungible_tokens" is not a self-describing naming. What exactly about FT do we store here? We don't store them as is (like `accounts` or `access_keys` tables), we seem to store updates/touches/changes (like `account_changes` table)

Reply:
```
aggregated__fungible_token_actions
aggregated__fungible_token_updates
aggregated__fungible_token_changes
aggregated__fungible_token_operations
```

I vote for `operations` and `changes`